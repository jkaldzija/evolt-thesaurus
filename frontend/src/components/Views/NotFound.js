import React from 'react';
import {Col, Row} from "reactstrap";
import SearchBox from "../SearchBox/SearchBox";
import AddWordView from "../AddWord/AddWordView";


// Set prop types
type NotFoundProps = {};

class NotFound extends React.Component<HomeProps> {
    componentDidMount() {
    }

    componentDidUpdate() {
    }

    render() {

        return (
            <Row>
                <Col col={12} className={"mb-4"}>
                    <div className={"d-flex flex-column justify-content-center align-items-center"}>
                        <h2 className={"typography-bold primary-text not-found"}>403</h2>
                        <div>Whoops, page not found</div>
                    </div>

                </Col>
            </Row>
        );
    };

}

// Set default props
NotFound.defaultProps = {};

export default NotFound;
