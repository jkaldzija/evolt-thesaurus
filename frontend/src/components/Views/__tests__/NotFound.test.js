import React from 'react';
import {shallow} from "enzyme";
import NotFound from "../NotFound";

describe('Not found', function () {
    it('should render correctly', function () {
        const component = shallow(
            <NotFound onSubmit={() => {}}/>
        );
        expect(component).toMatchSnapshot();
    });

});
