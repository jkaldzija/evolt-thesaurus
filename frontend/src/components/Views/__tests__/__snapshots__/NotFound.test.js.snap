// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Not found should render correctly 1`] = `
<Row
  tag="div"
>
  <Col
    className="mb-4"
    col={12}
    tag="div"
    widths={
      Array [
        "xs",
        "sm",
        "md",
        "lg",
        "xl",
      ]
    }
  >
    <div
      className="d-flex flex-column justify-content-center align-items-center"
    >
      <h2
        className="typography-bold primary-text not-found"
      >
        403
      </h2>
      <div>
        Whoops, page not found
      </div>
    </div>
  </Col>
</Row>
`;
